jiant/tasks/tasks.py

```diff
@@ -2776,10 +2769,10 @@ def make_one_hot(batch, depth=2):
             return ones.index_select(0, batch)
 
         binary_preds = make_one_hot(logits, depth=2)
-        # Make label_ints a batch_size list of labels
-        label_ints = torch.argmax(labels, dim=1)
-        self.f1_scorer(binary_preds, label_ints)
-        self.acc_scorer(binary_preds.long(), labels.long())
+        labels_one_hot = make_one_hot(labels, depth=2)
```

Comment:
why do we need to make things one-hot?

Reply:
Two other things:
1) take `make_one_hot` out of `update_metrics`? Seems like a weird place to define it
2) The usage of cuda on line 2767 will break if the user has a GPU available but decides to run on CPU, I think. `ones` will be a cuda tensor but everything else will be on GPU.

Reply:
We need to make this one hot because acc_scorer expects the labels and the predictions to be the same dimension. Let me try without one hot though!